reference/events.rst

```diff
@@ -80,6 +80,33 @@ Listener Class Name
 :class:`Symfony\\Bundle\\FrameworkBundle\\DataCollector\\RequestDataCollector`  0
 ==============================================================================  ========
 
+``kernel.controller_arguments``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 3.1
+     The ``kernel.controller_arguments`` event was introduced in Symfony 3.1.
+
+**Event Class**: :class:`Symfony\\Component\\HttpKernel\\Event\\FilterControllerArgumentsEvent`
+
+This event can be an entry point used to modify the arguments of the controller
+once they have been resolved by Symfony::
+
```

Comment:
you can put it where you want (as long as you tell Symfony that this method is a listener for this event) .

And the current proposal is consistent with other snippets in the page